app/src/main/java/com/appdev/eateryblueandroid/util/FavoritingUtils.kt

```diff
@@ -73,23 +51,27 @@ fun numFavorites(): Int {
     return num
```

Comment:
can we remove numFavorites method and getMutableFavoriteStateOf method?

Reply:
I will remove getMutableFavoriteStateOf in my next changes, but I feel that keeping numFavorites is necessary since otherwise I would have to change favoriteMap to public, which runs the risk of breaking its invariant (all values in favoriteMap must be equal to eatery's .isFavorite() for all eateries) if future additions tamper with it unknowingly.